xchange-core/src/main/java/org/knowm/xchange/dto/account/Balance.java

```diff
@@ -34,6 +36,7 @@
   private final BigDecimal borrowed;
   private final BigDecimal withdrawing;
   private final BigDecimal depositing;
+  private Date timestamp;
```

Comment:
Balance should be a value object

final fields, no setters

Reply:
fixed